Kernel/Devices/HID/HIDManagement.cpp

```diff
@@ -99,6 +99,11 @@ void HIDManagement::set_maps(NonnullOwnPtr<KString> character_map_name, Keyboard
     dbgln("New Character map '{}' passed in by client.", m_character_map_name);
 }
 
+void HIDManagement::attach_usb_hid_mouse_device(Badge<USB::Device>, MouseDevice& mouse)
+{
+    m_hid_devices.append(mouse);
+}
+
 UNMAP_AFTER_INIT ErrorOr<void> HIDManagement::enumerate()
 {
     // FIXME: When we have USB HID support, we should ensure that we disable
```

Comment:
This comment seems interesting

Reply:
I'd thought so too, but it's probably interesting on bare metal hardware on real UHCI controller. I'd need to look into this, because the UHCI controller in QEMU might allow us to disable the PS2 controller somehow (although they recently introduced a flag to [disable creation of PS/2 mouse and keyboard](https://lists.nongnu.org/archive/html/qemu-devel/2022-03/msg01279.html)).